pom.xml

```diff
@@ -64,9 +64,9 @@
     </developers>
 
     <properties>
-        <skipTests>true</skipTests>
+        <skipTests>false</skipTests>
```

Comment:
this is disable on purpose, because all test are currently integration tests against the cloud api (i.e., they need a valid api key)